tests/php/Extension/FluentFilteredExtensionTest.yml

```diff
@@ -0,0 +1,22 @@
+TractorCow\Fluent\Model\Locale:
+  nz:
+    ID: 1
+    Locale: en_NZ
+    Title: 'English (New Zealand)'
+    URLSegment: newzealand
+
+SiteTree:
+  home:
+    ID: 1
```

Comment:
I would recommend highly against hard-coded IDs even in tests.

If you found it challenging to express your logic purely in yml, you'll need to put your logic into setUp() instead.